recipes/typos/LICENSE-APACHE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
I don't think you need to add the licenses to the feedstock, as they are part of the repository. You just need to list them.